root/elections/ElectionDetails.js

```diff
@@ -77,7 +77,11 @@ const ElectionDetails = ({election, user}: PropsT) => (
           {election.is_open
             ? (
               lp(election.status_name, 'autoeditor election status', {
-                date: formatUserDate(user, election.open_time),
+                date: (
+                  election.open_time
+                    ? formatUserDate(user, election.open_time)
+                    : l('an unknown date')
+                ),
```

Comment:
I'm not sure there's ever a case where election.open_time doesn't exist if there's election.is_open, but I guess it doesn't hurt to make sure anyway? (it's optional in types.js)

Reply:
Yeah, it'll always be defined if `is_open` is true (it's only undefined if still waiting for seconders). I think there's a way to express that to Flow using intersection/union types, but having the check seems fine for now. I'd probably output `null` in the else case instead of making people translate a string that'll never be displayed though. :)

Reply:
I just feel it's weird to even have the chance to display "Voting open since " (and nothing else), but I guess it if never happens that's ok. Just strange :)

Reply:
That's the thing, there is no chance. :P It's easy to prove that [on the server](https://github.com/metabrainz/musicbrainz-server/blob/962c5c01e64988b8197eb6ea1325bec6af69ff7f/lib/MusicBrainz/Server/Data/AutoEditorElection.pm#L102-L104) but we just need to convince Flow of it somehow later.

Reply:
Ok then, updated.